src/auth-service/models/Inquiry.js

```diff
@@ -23,6 +23,14 @@ const InquirySchema = new mongoose.Schema(
       required: [true, "fullName is required!"],
       trim: true,
     },
+    firstName: {
```

Comment:
Adding firstName and lastName to support recent Figma designs for the part of GET INVOLVED. @BenjaminSsempala